### PR TITLE
More doc improvements.

### PR DESCRIPTION
**Pull request by cmarqu**
This also adds a few FIXME docstrings in ``trigger.py`` that I cannot confidently handle myself, but I feel having these methods show up in the documentation in the first place is even more important.
I very much welcome docstring suggestions of course.
In the ``examples.rst`` there are still some small TODOs, this will come in a later PR.

### CODE REVIEW

File: cocotb/bus.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-# Copyright (c) 2013 Potential Ventures Ltd
+# Copyright (c) 2013, 2019 Potential Ventures Ltd
```
Comment (imphil) [inline cocotb/bus.py]: Please don't update the copyright year. I've opened #804 to see how we can fix the headers in the longer run.

File: cocotb/drivers/__init__.py [hunk 2]
```diff
@@ -96,6 +102,7 @@ def __init__(self):
         self._thread = cocotb.scheduler.add(self._send_thread())
 
     def kill(self):
+        """Kill the coroutine sending stuff."""
```
Comment (imphil) [inline cocotb/drivers/__init__.py]: Maybe replace "stuff" with something more formal and specific?

File: cocotb/drivers/__init__.py [hunk 3]
```diff
@@ -130,7 +137,7 @@ def send(self, transaction, sync=True, **kwargs):
 
         Args:
             transaction (any): The transaction to be sent.
-            sync (bool,  optional): Synchronise the transfer by waiting for rising edge.
+            sync (bool, optional): Synchronise the transfer by waiting for rising edge.
```
Comment (imphil) [inline cocotb/drivers/__init__.py]: nit: "for _a_ rising edge" or "for _the next_ rising edge"

Comment (cmarqu) [reply cocotb/drivers/__init__.py]: Done.

File: cocotb/drivers/__init__.py (hunk 2 above)
Comment (cmarqu) [reply cocotb/drivers/__init__.py]: The "stuff" is actually taken from a nearby comment. I don't feel I have the knowledge to make this better without potentially introducing inaccuracies.

**Comment by imphil** [comment]
Thanks @cmarqu, great stuff!

**Comment by cmarqu** [comment]
"stuff", I see what you did there! :)

**Pull request merged**